src/devices/Mcp3428/Mcp3428.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!--Disabling default items so samples source won't get build by the main library-->
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="*.cs" />
+    <None Include="README.md" />
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
```

Comment:
recently we changed all Version to `$(SystemDeviceGpioPackageVersion)`, see i.e.: https://github.com/dotnet/iot/blob/master/src/devices/Bmp180/Bmp180.csproj#L13

Reply:
So is it no longer possible to build the project standalone?  
Makes it inconvenient to develop devices, especially when you have the project included in some of your own projects.

Reply:
I also came across this issue and didn't realize.  I usually create my binding projects outside of the iot structure at first.  This new property makes us have to remember to update before creating a PR.

Reply:
You can always have your own project outside of the repo to define this property as well. There are two main reasons why we want to keep the property instead of the wildcard version:
 - You make sure that you reference the last produced build, as opposed to maybe a feed being down or a version to be broken and we silently get what ever version NuGet resolves without us noticing it.
 - Most importantly, it enables us to be able to have deterministic builds to any point in time. If today you do a git checkout and go back in time to when we used the wildcard notation, you will find that you won't be able to build. This is because when ingesting new versions, it may require changes in the repo side in order to get those versions working, and when using wildcards, you will always get the latest even though the repo might not have the required changes yet in order to consume this version.